src/components/text/Heading.tsx

```diff
@@ -5,6 +5,8 @@ export const Heading = styled.Text<{ centered?: boolean }>`
   font-family: ${props => props.theme.fonts.contentFontBold};
   font-weight: ${props => props.theme.fonts.defaultFontWeight};
   ${props => (props.centered ? 'align-self: center;' : '')};
+  text-align: center;
+  margin-bottom: 20px;
```

Comment:
this will set the margin for all headings. Just change the relevant heading in `ManageSelectionsScreen`